client/pixelboards.js

```diff
@@ -272,17 +271,16 @@
                 return;
             }
 
+            var colorHistory = pixel ? pixel.color : color;
+            var actionType = pixel ? self.actionsType[2] : self.actionsType[0];
+            self.history.add(
```

Comment:
You need to keep the test with `addToHistory` boolean for the history to work normally.